ts/test/scenario.test.ts

```diff
@@ -499,30 +594,113 @@ test("runScenario - add players and then install apps for them", async () => {
-    const players = await scenario.addPlayers(2);
-    const [aliceApp, bobApp] = await scenario.installSameAppForPlayers(
-      { appBundleSource: { type: "path", value: FIXTURE_HAPP_URL.pathname } },
-      players,
-    );
-    assert.deepEqual(aliceApp.agentPubKey, players[0].agentPubKey);
-    assert.deepEqual(bobApp.agentPubKey, players[1].agentPubKey);
-
-    const content = "test-content";
-    const createEntryHash = await aliceApp.cells[0].callZome<EntryHash>({
-      zome_name: TEST_ZOME_NAME,
-      fn_name: "create",
-      payload: content,
-    });
+test(
+  "runScenario - add players and then install the same app for them",
+  async () => {
+    await runScenario(async (scenario) => {
+      const players = await scenario.addPlayers(2);
+      const [aliceApp, bobApp] = await scenario.installSameAppForPlayers(
+        { appBundleSource: { type: "path", value: FIXTURE_HAPP_URL.pathname } },
+        players,
+      );
+      assert.deepEqual(aliceApp.agentPubKey, players[0].agentPubKey);
+      assert.deepEqual(bobApp.agentPubKey, players[1].agentPubKey);
+
+      const content = "test-content";
+      const createEntryHash = await aliceApp.cells[0].callZome<EntryHash>({
+        zome_name: TEST_ZOME_NAME,
+        fn_name: "create",
+        payload: content,
+      });
 
-    await dhtSync([aliceApp, bobApp], aliceApp.cells[0].cell_id[0]);
+      await dhtSync([aliceApp, bobApp], aliceApp.cells[0].cell_id[0]);
 
-    const readContent = await bobApp.cells[0].callZome<typeof content>({
-      zome_name: TEST_ZOME_NAME,
-      fn_name: "read",
-      payload: createEntryHash,
+      const readContent = await bobApp.cells[0].callZome<typeof content>({
+        zome_name: TEST_ZOME_NAME,
+        fn_name: "read",
+        payload: createEntryHash,
+      });
+      assert.equal(readContent, content);
     });
-    assert.equal(readContent, content);
-  });
-});
+  },
+  {
+    timeout: 100000,
+  },
+);
+
+test(
+  "runScenario - 0-arc conductor",
+  async () => {
+    await runScenario(async (scenario) => {
+      const alice = await scenario.addPlayerWithApp({
+        appBundleSource: {
+          type: "path",
+          value: FIXTURE_HAPP_URL.pathname,
+        },
+        options: {
+          networkConfig: {
+            targetArcFactor: 0,
+          },
+        },
+      });
+
+      const bob = await scenario.addPlayerWithApp({
+        appBundleSource: {
+          type: "path",
+          value: FIXTURE_HAPP_URL.pathname,
+        },
+      });
+
+      const sue = await scenario.addPlayerWithApp({
+        appBundleSource: {
+          type: "path",
+          value: FIXTURE_HAPP_URL.pathname,
+        },
+      });
+
+      // Bob creates an entry
+      const createEntryHash = await bob.cells[0].callZome<EntryHash>({
+        zome_name: TEST_ZOME_NAME,
+        fn_name: "create",
+        payload: "test-content",
+      });
+
+      // Wait for bob & sue to sync, so that bob declares a full arc
```

Comment:
Having the same DHT content does not guarantee a full arc. That will happen in the next gossip round after the sync completes. That's configured to run fast in Tryorama but it won't be entirely reliable to use `dhtSync` to have a peer to make get requests to.